invenio_app_ils/eitems/mappings/os-v1/eitems/eitem-v1.0.0.json

```diff
@@ -61,6 +61,9 @@
       "document_pid": {
         "type": "keyword"
       },
+      "eitem_type": {
```

Comment:
do we need the prefix `eitem_`, given that we are already in the e-item?

Reply:
My reasoning for the prefix `eitem_` was to be consistent with naming like `document_type` for documents. Wouldn't it be confusing with just type?